### PR TITLE
Update showdown to fix mem vulnerability

### PR DESCRIPTION
**Pull request by lordelix**
## Description
Updates the `showdown`  package minor version, to get rid of the `mem@1` dependency, which has a vulnerability.

See --> https://github.com/owncloud/core/pull/36752/files#diff-f1a9d00aaecb7a224613fd9e4312f71eL1598

## How Has This Been Tested?
CI

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:
- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [x] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

**Comment by micbar** [comment]
looks good. Please add changelog item.

**Comment by micbar** [comment]
@felixheidecke We need that on the `release-10.4.0` branch

**Comment by felixheidecke** [comment]
@micbar here goes the changelog item https://github.com/owncloud/core/pull/36763

**Comment by phil-davis** [comment]
why not push the changelog commit onto this PR?

**Comment by felixheidecke** [comment]
@phil-davis ... hmmm ... that would have been the better way :-| ... *sigh*

@micbar https://github.com/owncloud/core/pull/36765

**Comment by felixheidecke** [comment]
I can cherry-pick the changes into this and the relelease branch PR if that makes things easier?!

**Comment by phil-davis** [comment]
> I can cherry-pick the changes into this and the relelease branch PR if that makes things easier?!

yes, include the changelogs in the actual PRs and close the other "changelog-only" PR. That is the "normal" practice (this is changelog stuff is a bit new, so I guess people are not all familiar with "normal" just yet)

And @micbar can decide if just the PR to the release branch, or both PRs get merged.

**Comment by felixheidecke** [comment]
@phil-davis I picked the pocket! 😜

**Comment by micbar** [comment]
@felixheidecke please rebase.

**Comment by felixheidecke** [comment]
Done.

File: changelog/unreleased/36752 [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Change: Update handlebars library to 4.5.3
```
Comment (phil-davis) [inline changelog/unreleased/36752]: what does this have to do with handlebars 4.5.3 ?

Comment (felixheidecke) [reply changelog/unreleased/36752]: Absolutely nothing ... was a copy & paste mistake 😅

**Review by phil-davis — APPROVED** [review]
